Source/WebCore/rendering/AttachmentLayout.mm

```diff
@@ -135,7 +135,7 @@ static Color titleTextColorForAttachment(const RenderAttachment& attachment, Att
 
 void AttachmentLayout::layOutSubtitle(const RenderAttachment& attachment)
 {
-    auto& subtitleText = attachment.attachmentElement().attributeWithoutSynchronization(HTMLNames::subtitleAttr);
+    String subtitleText = attachment.attachmentElement().attachmentSubtitleForDisplay();
```

Comment:
Can leave as `auto&`.

Reply:
Nope, can't do in this case: `attributeWithoutSynchronization` used to return a `const AtomString&`, but now `attachmentSubtitleForDisplay` returns a `String` temporary object, which cannot be bound to a non-const lvalue reference.
`const auto&` would work, if you preferred?

Reply:
Looks like `String` is used in other call sites, so this is fine.